src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -272,51 +279,24 @@ private ReviewModel CreateNewReview(PullRequestModel prModel)
                     var baseline = await CreateBaselineRevision(baselineCodeFile, baseLineStream, pullRequestModel, baselineFileName);
                     review.Revisions.Add(baseline);
                 }
-                var reviewUrl = REVIEW_URL.Replace("{hostName}", hostName).Replace("{ReviewId}", review.ReviewId);
-                stringBuilder.Append($"API review is created for `{codeFile.PackageName}`. You can review APIs [here]({reviewUrl}).").Append(Environment.NewLine);
             }
             else
             {
                 // Check if API surface level matches with any revisions
                 var renderedCodeFile = new RenderedCodeFile(codeFile);
                 if (await IsReviewSame(review, renderedCodeFile))
                 {
-                    //Do not update the comment if review was already created and it matches with current revision.
-                    if (pullRequestModel.ReviewId != null)
-                        return "";
-
-                    //Baseline review was not created earlier or this is the first commit of PR
-                    stringBuilder.Append($"API changes are not detected in this pull request for `{codeFile.PackageName}`");
-                    return stringBuilder.ToString();
+                    return;

```

Comment:
Do we need to store this info so we can add it to the aggregate comment?

Reply:
Can we not add unmodified packages to the single comment and with just a note that there are no changes? Or maybe even a link to the diff like we talked about with SDK automation where it is a link to an API diff that has no changes.

Reply:
@weshaggard : I have created an issue for tracking of this change to show unchanged packages list. https://github.com/Azure/azure-sdk-tools/issues/3348 

Can I go ahead and merge this PR?